### ISSUE TITLE
Confirm/update installation instructions for OS X

### ISSUE THREAD

**Issue by stomar** [ruby/www.ruby-lang.org#915]
On the `installation` page, the "Homebrew (OS X)" section needs confirmation or update.

 _Can anyone confirm?_

**Comment by joneslee85** [comment]
@stomar 

```
brew info ruby
ruby: stable 2.1.5 (bottled), HEAD
https://www.ruby-lang.org/
```

**Comment by JuanitoFatas** [comment]
Yosemite shipped with `ruby 2.0.0p481 (2014-05-08 revision 45883) [universal.x86_64-darwin14]`

**Comment by stomar** [comment]
I see. Thanks!
